controller/testTCPserver.py

```diff
@@ -0,0 +1,28 @@
+import unittest 
+from tcpServer import *
+import socket 
+from testingClass import TestClass 
+
+# in order to run tests, run this file and then run mockClient.py
```

Comment:
Could make a README.md for this folder and put these details in there (e.g., under a section called "Usage").